cifar10-multiannotator/utils/model_training.py

```diff
@@ -0,0 +1,272 @@
+import sys
```

Comment:
Github won't let me comment on multiannotator_labels.ipynb notebook because it says "the diff is too large" (51,011 additions)...  So I'll put my comments about your notebook in this area instead, sorry if it's not the most straightforward.

The reason the notebook is so huge is because autogluon is printing too much stuff during `fit()`, which makes the output cell of your iterative-training block humongous.  You could either suppress the output of this cell, or call `ImagePredictor.fit(..., verbosity=0)`

Reply:
Also seems like that iterative block only stopped because you imposed a `KeyboardInterrupt`?  I think we should apply max number of iterations (say 5 or 10).

Reply:
This was pre testing for optimal max itterations but resolved now :D

Reply:
@jwmueller I would rather minimize verbosity but "verbosity" does not seem to be a hypterparam of .fit() https://auto.gluon.ai/dev/api/autogluon.predictor.html#autogluon.vision.ImagePredictor.fit

I already set ImagePredictor.verbosity=0

Reply:
Oh ya I see you already set verbosity = 0, seems to be a bug that it's not suppressing output...

I'm not sure there's any other way to reduce output, so you may want to just suppress this cell, or you can try redirecting the autogluon output to file if you can figure out some way to do that for only one Python command but not others in the same code loop.